src/main/java/org/springframework/data/couchbase/core/convert/OtherConverters.java

```diff
@@ -187,4 +197,36 @@ public Class<?> convert(String source) {
 		}
 	}
 
+	/**
+	 * Writing converter for Enums. This is registered in
+	 * {@link org.springframework.data.couchbase.config.AbstractCouchbaseConfiguration#customConversions( CryptoManager)}.
+	 * The corresponding reading converters are in {@link IntegerToEnumConverterFactory} and
+	 * {@link StringToEnumConverterFactory}
+	 */
+
+	@WritingConverter
+	public static class EnumToObject implements Converter<Enum<?>, Object> {
+		ObjectMapper objectMapper;
```

Comment:
no.  AbstractCouchbaseConfiguration.java:431
			configurationAdapter.registerConverter(new OtherConverters.EnumToObject(getObjectMapper()));
			
I did not figure out how to pass in this objectMapper into this block in CouchbaseCustomConversions where the static OtherConverters.getConvertersToRegister()) is called, otherwise it could be private.  (Also, all the other converters in OtherConverters are 'public' (although they do not need to be).)

	static {

		List<Object> converters = new ArrayList<>();

		converters.addAll(DateConverters.getConvertersToRegister());
		converters.addAll(CouchbaseJsr310Converters.getConvertersToRegister());
		converters.addAll(OtherConverters.getConvertersToRegister());

		STORE_CONVERTERS = Collections.unmodifiableList(converters);
		STORE_CONVERSIONS = StoreConversions.of(SimpleTypeHolder.DEFAULT, STORE_CONVERTERS);
	}

Reply:
I misunderstood.  I thought the comment was for the method on the line above instead of the object indicated. Fixed.